### PR TITLE
Bring get_network_base up to package level

### PR DESCRIPTION
**Pull request by ktbyers**
Also minor cleanup so pylama passes.

### CODE REVIEW

File: napalm/__init__.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import pkg_resources
 import sys
+from napalm.base import get_network_driver
```
Comment (dbarrosop) [inline napalm/__init__.py]: shouldn't we just move the code?

File: requirements/dev [hunk 2]
```diff
@@ -1,3 +1,4 @@
+future
```
Comment (dbarrosop) [inline requirements/dev]: this is in the `base` requirements already, do you think we need it here too?

Comment (ktbyers) [reply requirements/dev]: Will remove...missed it there.

File: napalm/__init__.py (hunk 1 above)
Comment (ktbyers) [reply napalm/__init__.py]: It shouldn't matter...napalm.base.__init__ will be processed regardless (as get_network_driver will use code from napalm.base).

**Pull request merged**